source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -79,7 +80,7 @@ class MessageSerializer(PbNestedSerializer):
 
 class SeveritySerializer(PbNestedSerializer):
     name = jsonschema.Field(schema_type=str)
-    effect = jsonschema.Field(schema_type=str)
+    effect = EnumField(attr='effect', pb_type=Severity.Effect, lower_case=False)
```

Comment:
Same here, is `attr` necessary?

Reply:
It's not necessary but i added that to distinct from stop_time_effect in my modifications